_site/css/application.css

```diff
@@ -245,6 +245,38 @@ hr {
       background-color: #000;
       color: #fbcfc6; }
 
+div.social {
+  background-color: pink;
+  float: left; }
+  div.social i {
+    color: #d8ae64; }
+    div.social i:hover {
+      color: #fff; }
+
+div.social_links {
+  margin: auto;
+  width: 50%;
+  text-align: center; }
+  div.social_links i {
+    color: red; }
+    div.social_links i:hover {
```

Comment:
this hover effect is working a little strangely and I'm not sure it's applied to the correct element. The yellow background color hover gets applied when you hover just outside the icon like this:
![image](https://user-images.githubusercontent.com/12739849/37235809-e6f85c46-23cf-11e8-918a-fedfb1dcde61.png)

But the icon color doesn't change to white until you actually hover on the icon.